packages/flutter/lib/src/services/clipboard.dart

```diff
@@ -59,7 +59,7 @@ abstract final class Clipboard {
     return ClipboardData(text: result['text'] as String);
   }
 
-  /// Returns a future that resolves to true iff the clipboard contains string
```

Comment:
Might this be 'if and only if'? https://www.dictionary.com/browse/iff

Reply:
+1, to avoid future confusion we could just spell this out:

```suggestion
  /// Returns a future that resolves to true if and only if the clipboard contains string
```

Reply:
Hi @Amir-P it looks like the github UI doesn't allow accepting suggestion from deleted line. Can you update your branch with the suggestion?